README.md

```diff
@@ -13,61 +13,118 @@
+
+If you use Prometheus and are also familiar with PostgreSQL, then Promscale is a great choice for
+your Prometheus remote storage. You can scale to millions of series and hundreds of thousands of samples
+on a single PostgreSQL node thanks to TimescaleDB.
+
+To get started:
+1. [Install Promscale](https://docs.timescale.com/promscale/latest/installation/#install-promscale-with-instrumentation).
+2. [Configure Prometheus](https://docs.timescale.com/promscale/latest/send-data/prometheus/) to send data to Promscale.
+3. [Configure Grafana](https://docs.timescale.com/promscale/latest/visualize-data/grafana/) to query and visualize metrics from Promscale
+using a PromQL and/or a PostgreSQL datasource.
+
+## Promscale for Jaeger and OpenTelemetry
+
+Promscale can natively ingest OpenTelemetry traces and it can also ingest Jaeger and Zipkin traces via
+the OpenTelemetry Collector.
+
+Promscale solves three main use cases for Jaeger and OpenTelemetry users:
+
+* **Easy-to-use durable and scalable storage backend for traces**<br/>
+Most users run Jaeger with the in memory or badger storage because the two options for a more durable storage
+(Elasticsearch and Cassandra) are difficult to set up and operate. Promscale uses a much simpler architecture
+based on PostgreSQL which many developers are comfortable with and scales to 100s of thousands of spans per
+second on a single database node.
+
+* **Service performance analysis**<br/>
+Because Promscale can store both metrics and traces, you can use the new 
+[Service Performance Management](https://www.jaegertracing.io/docs/1.36/spm/) feature in Jaeger with Promscale 
+as the only storage backend for the entire experience.
+Promscale also includes a fully customizable, out-of-the-box, and modern
+[Application Performance Management (APM) experience](https://docs.timescale.com/promscale/latest/visualize-data/apm-experience/)
+in Grafana built using SQL queries on traces.
 
-For a detailed description of the initial architecture of Promscale for Prometheus metrics that covers
-some key design principles, please see [our design doc](https://tsdb.co/prom-design-doc).
+* **Trace analysis**<br/>
+Jaeger searching capabilities are limited to filtering individual traces. This is helpful when troubleshooting problems once you know
+what you are looking for. With Promscale you can use SQL to interrogate your trace data in any way you want and discover issues
+that would usally take you a long time to figure out by just looking at log lines, metric charts or individual traces. You can see some
+examples in [this blog post](https://www.timescale.com/blog/what-are-traces-and-how-sql-yes-sql-and-opentelemetry-can-help-us-get-more-value-out-of-traces-to-build-better-software/)
+
+**Key features:** native OTLP support, SQL queries, APM capabilities, data compression, data retention
+
+**Try it out** by installing our lightweight [opentelemetry-demo](https://github.com/timescale/opentelemetry-demo) with a single
```

Comment:
Do we have a demo for prometheus?

Reply:
We do not have one, Usually, if users deploy Prometheus it already has its own metrics as a bonus they can also scrape metrics from Promscale Connector to ingest some metric data into Promscale.

Reply:
We should have one though. The general getting started that Vineeth is putting together could be a good starting point and if we see many people use it then we can look into offering a more specific and detailed one.